cpp/include/cudf/transform.hpp

```diff
@@ -83,5 +83,29 @@ std::pair<std::unique_ptr<rmm::device_buffer>, size_type> nans_to_nulls(
 std::pair<std::unique_ptr<rmm::device_buffer>, cudf::size_type> bools_to_mask(
   column_view const& input, rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Encode the values of the given column as integers
+ *
+ * The encoded values are integers in the range [0, n), where `n`
```

Comment:
Describe the behavior for nulls in `input`.

Reply:
Just added a description - let me know if it's clear enough

Reply:
Ah but it's not really accurate.

Reply:
Fixed